app/assets/javascripts/application.js.erb

```diff
@@ -636,9 +636,11 @@ onPageLoad(() => {
   });
 
   on('click', 'a.story-disowner', (event) => {
+    event.preventDefault();
+
     if (confirm("Are you sure you want to disown this story?")) {
       let li = parentSelector(event.target, '.story');
-      fetchWithCSRF('/stories/' + li.getAttribute('data-shortid') + '/disown', { method: 'post' })
+      fetchWithCSRF(event.target.getAttribute("href"), { method: 'post' })
```

Comment:
I really like how this change means the js reuses the existing URL generation instead of reimplementing it.